src/main/proto/ga4gh/assay_metadata.proto

```diff
@@ -0,0 +1,96 @@
+syntax = "proto3";
+
+package ga4gh;
+
+import "google/protobuf/struct.proto";
+
+
+// An experimental preparation of a sample.
+message Experiment {
+  // The experiment UUID. This is globally unique.
+  string id = 1;
+
+  // The name of the experiment.
+  string name = 2;
+
+  // A description of the experiment.
+  string description = 3;
+
+  // The time at which this message was created.
+  // Format: ISO 8601, YYYY-MM-DDTHH:MM:SS.SSS (e.g. 2015-02-10T00:03:42.123Z)
+  string message_create_time = 4;
```

Comment:
Why message_create_time here? The work ``message`` seems ambigous, i.e., does this refer to the individual  message objects as they are being sent to the client, or to the 'record' that is stored on the server? I'm guessing that this is because of a clash with ``run_time``? I would have thought it would be better to just use ``created`` and ``updated`` just for the sake of being consistent with all the other objects (and change them all in one go to something more specific like ``creation_timestamp``, ``update_timestamp``, if/when this is done).

I realise this happened before this specific set of changes, so please feel free to point me in the direction of the relevant discussion. This isn't a criticism of the present PR, I just thought I'd bring it up while we're reorganising this metadata stuff.